changelog/fix-9585-rename-instant-payouts

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: update
```

Comment:
Not sure if only dev or it can be any value here so that changelog is 'ignored'.

```suggestion
Type: dev
```

Reply:
I think the type can be `update` .. It should be `patch` and have a `Comment` . The PR also delves into merchant-facing UI and hence should be `update` instead of `dev`.